### PR TITLE
test(math): reinforce test for parallel IFFT

### PR DESCRIPTION
**Pull request by TomTaehoonKim**
# Description

The test degree was too low to test the parallel part of IFFT. This PR reinforces IFFT test by increasing the degree.

### CODE REVIEW

File: tachyon/math/polynomials/univariate/univariate_evaluation_domain_unittest.cc [hunk 1]
```diff
@@ -256,7 +256,7 @@ TYPED_TEST(UnivariateEvaluationDomainTest, FFTCorrectness) {
   using DensePoly = typename Domain::DensePoly;
   using Evals = typename Domain::Evals;
 
-  const size_t log_degree = 5;
+  const size_t log_degree = 7;
```
Comment (chokobole) [inline tachyon/math/polynomials/univariate/univariate_evaluation_domain_unittest.cc:259]: Could you change this when `TACHYON_HAS_OPENMP` is on?

**Review by chokobole — APPROVED** [review]
LGTM

**Review by Insun35 — APPROVED** [review]
LGTM

**Review by ashjeong — APPROVED** [review]
LGTM

**Pull request merged**